docs/flow-setup-tutorial.md

```diff
@@ -0,0 +1,34 @@
+# Flow setup Tutotrial
+
+Flow is a static-type checker for Javascript code
+
+## The steps to install Flow correctly from scratch are
+
+####  Installation tip : install flow in the root folder of zulip-mobile
+
+####  package installer used - yarn
+
+## Installing flow steps
+
+1. install babel compiler with command line interface,
+	run command <$yarn add --dev babel-cli babel-preset-flow>
+2. change the present from "react-native" to "flow"
+>	{
+>	"present": ["flow"]
+>	}
+3. Install flow version 0.65 for zulip using <$yarn add --dev flow-bin@0.xx> where xx is the flow version defined in .flowconfig file.
+4. type <$flow init> 
```

Comment:
This is not needed. This is setting up Flow in a new project. We have already done that.